sources/packages/backend/apps/api/src/app.students.module.ts

```diff
@@ -55,6 +54,7 @@ import { LoggerModule } from "./logger/logger.module";
 import { ApplicationControllerService } from "./route-controllers/application/application.controller.service";
 import { StudentAccountApplicationsService } from "./services/student-account-applications/student-account-applications.service";
 import { ATBCStudentController } from "./route-controllers/atbc/atbc.students.controller";
+import { WorkflowClientService, SequenceControlService } from "@sims/services";
```

Comment:
Why is the workflow-client-service created in the shared library? I was assuming this client service will be used only by our API. Is it keeping the future perspective in mind that something other than API might also need this client service in future?

Reply:
The `WorkflowClientService` contains interactions with the workflow that includes starting workflows, sending messages, canceling, and more. This type of interaction can happen from the API or from workers. We are not using it right now for workers but I see that possibly being used in the future.

Reply:
I am just trying to understand what could be the possible scenario of a worker trying to use one of these client services.(starting an instance, cancelling and sending message)